test/e2e/manifest/postgres.yaml

```
@@ -0,0 +1,92 @@
+            - cruisekube
+          initialDelaySeconds: 5
+          periodSeconds: 5
+      volumes:
+      - name: init-script
+        configMap:
+          name: postgres-init-script
+  volumeClaimTemplates:
+  - metadata:
+      name: postgres-storage
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)